week1/homework/src/server.js

```diff
@@ -5,16 +5,44 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+function handleRequest(response, state) {
+  response.writeHead(200, { 'content-type': 'application/json' });
```

Comment:
It is great that you have put this logic in a separate function and reused it 👍

Reply:
Thanks! :)